code42.json

```diff
@@ -629,6 +629,331 @@
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.alert_state",
+                    "data_type": "string",
+                    "column_name": "Alert State",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.start_date",
+                    "data_type": "string",
+                    "column_name": "Start Date",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.parameter.end_date",
+                    "data_type": "string",
+                    "column_name": "End Date",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.name",
+                    "data_type": "string",
+                    "column_name": "Alert Name",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.data.severity",
+                    "data_type": "string",
+                    "column_name": "Severity",
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.data.description",
+                    "data_type": "string",
+                    "column_name": "Description",
+                    "column_order": 7
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "set alert state",
+            "identifier": "set_alert_state",
+            "description": "Set the state of the alert",
+            "verbose": "",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "alert_id": {
+                    "description": "Alert ID",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
```

Comment:
If you are going to use the `alert` contains like you did above for the `alert ID`, it should be used here as well (and everywhere `alert id` is defined